src/consts.h

```diff
@@ -37,6 +37,7 @@ enum class ImuID {
     BMI160,
     ICM20948,
     ICM42688,
+    BMI220,
```

Comment:
Please put new sensor at the end, regardless of alphabetical order, server expects particular ID.

Reply:
Sorry for interrupting, I will change it later